src/main/java/com/dawkastresu/medicalclinic/PatientController.java

```diff
@@ -16,32 +16,37 @@ public class PatientController {
     private final PatientService patientService;
 //pusta adnotacja GetMapping oznacza endpoint /patients, w którym obslugiwana jest metoda GET, czyli zwrócenie zasobu
     @GetMapping
-    public List<Patient> getAllPatients() {
-        return patientService.getAllPatients();
+    public List<PatientDto> getAllPatients() {
+        return patientService.getAll();
     }
 //GetMapping/email oznacza endpoint /patient/email gdzie email jest zmienną ścieżkową oraz obslugiwana jest metoda HTTP GET, czyli zwrócenie zasobu
     @GetMapping("/{email}")
-    public Patient getPatientByMail(@PathVariable String email) {
+    public PatientDto getPatientByMail(@PathVariable String email) {
         return patientService.findPatientByName(email);
     }
 //ResponseStatus okresla status zwracany w odpowiedzi http - 201 CREATED
     @ResponseStatus(HttpStatus.CREATED)
 //PostMapping określa enpoint /patients dla metody http POST - czyli dodanie nowego zasobu
     @PostMapping
     public Patient addNewPatient(@RequestBody Patient patient) {
```

Comment:
Przejdźmy na konwencję Command jak mam utworzyć pacjenta to zrób klasa CreatePatientCommand